third_party/proto/cosmos_proto/cosmos.proto

```diff
@@ -13,4 +13,6 @@ extend google.protobuf.MessageOptions {
 
 extend google.protobuf.FieldOptions {
     string accepts_interface = 93001;
+
+    string scalar = 93002;
```

Comment:
Let's open an PR on cosmos-proto with this file

Reply:
Sure. I will take of it

Reply:
Done

Reply:
Oh sorry, I should have been more clear: it should be the https://github.com/cosmos/cosmos-proto repo

Reply:
My bad! I was suggesting Andrei to do this on the regen repo as that's where the file is defined at the moment.... and wasn't sure if you meant just adding the `scalar` field option to the existing proto package.